IdentityCore/src/oauth2/aad_v1/MSIDAADV1TokenResponse.m

```diff
@@ -24,9 +24,20 @@
 #import "MSIDAADV1TokenResponse.h"
 #import "MSIDAADV1IdTokenClaims.h"
 #import "MSIDTokenResponse+Internal.h"
+#import "MSIDJsonSerializableTypes.h"
+#import "MSIDJsonSerializableFactory.h"
 
 @implementation MSIDAADV1TokenResponse
 
++ (void)load
+{
+    if (@available(iOS 13.0, *))
```

Comment:
do we also want to disable this code in ADAL?

Reply:
Agreed to remove "if (@available(iOS 13.0, *))"